options/locale/locale_en-US.ini

```
@@ -1263,10 +1263,12 @@ issues.new.no_assignees = No Assignees
 issues.new.no_reviewers = No reviewers
 issues.new.add_reviewer_title = Request review
 issues.choose.get_started = Get Started
+issues.choose.open_external_link = Open
 issues.choose.blank = Default
 issues.choose.blank_about = Create an issue from default template.
 issues.choose.ignore_invalid_templates = Invalid templates have been ignored
 issues.choose.invalid_templates = %v invalid template(s) found
+issues.choose.invalid_config = Your issue_config.yaml contains a error:
```

```suggestion
issues.choose.invalid_config = The issue config contains errors:
```

IMO:

- Not `your`, the issue poster may do not own the repo.
- It's not `issue_config.yaml`.